compiler/aarch64/codegen/OMRMemoryReference.hpp

```diff
@@ -60,6 +60,8 @@ class OMR_EXTENSIBLE MemoryReference : public OMR::MemoryReference
    TR::Node *_indexNode;
    int32_t _offset;
 
+   TR::Register *_modBase;
```

Comment:
Can you explain what `_modBase` is used for?  One-third of your assignment logic is dedicated to assigning it, but its not immediately clear to me what its for.

Reply:
Yes, unless you can explain what it will be used for in AArch64.